Tests/Python/PythonPackagesTests.cs

```diff
@@ -826,6 +826,37 @@ public void CopulaeTest()
     return cop.summary()"
             );
         }
+                [Test]
+        public void MonoTest()
```

Comment:
Hey @aarjaneiro! Thanks for the test, only comment is I think it needs a more precise name, I'm not really good with names 😅 but maybe something like: `SanityClrInstallation`?

Reply:
Good idea! I have pushed SanityClrInstallation as a name directly onto my master branch.

Reply:
Thanks!